supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -13,6 +13,16 @@
 // Fuse: Prefixed second sentence with "In Red Hat Fuse, an EIP is a" (Breda)
 // Fuse: Changed "Camel" to "Red Hat Fuse" (Breda)
 // Fuse: Removed the final sentence about the Palette (Breda)
+
+[[egress]]
+==== image:images/yes.png[yes] egress (adjective, noun)
+*Description*: In RedHat OpenShift, _egress_ describes the outgoing direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
```

Comment:
```suggestion
*Description*: In Red{nbsp}Hat OpenShift, _egress_ describes the process of network traffic leaving an OpenShift cluster. Data _ingress_, on the other hand, refers to outside network traffic that enters an OpenShift cluster.
```
- Consider this revision as a way of simplifying the contrast between _egress_ and _ingress_.
- Add non-breaking space code between Red and Hat.

Reply:
Thanks, Brian. Please swap "other hand" with one of the following IBM SG suggestions:

![Uploading Screenshot from 2024-03-20 09-24-02.png…]()